package.json

```diff
@@ -39,10 +39,10 @@
     "supertest": "1.2.0"
   },
   "scripts": {
-    "pretest": "npm run lint",
+    "posttest": "npm run lint",
     "test": "nyc _mocha -b",
     "mocha": "_mocha -b",
-    "lint": "eslint .",
+    "lint": "eslint . --max-warnings 0",
```

Comment:
I don't think this is doable with .eslintrc (I already lost a lot of time on this on another project), If you know how to do it please let me know (but don't waste a day on this ;=)